performance-test/tasks/notification_tasks.py

```
@@ -1,15 +1,54 @@
+                            self.client.patch("/api/notifications",
+                                              headers=headers,
+                                              name="/api/notifications (All)")
+                        else: # 개별 확인
+                            target_id = random.choice(unread_list).get("id")
+                            self.client.patch(f"/api/notifications/{target_id}",
+                                              headers=headers,
+                                              name="/api/notifications/{id}")
+                    else:
+                        # 읽을 알림이 없으면 새 유저로 교체
+                        self.pick_new_user()
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)